### DISCUSSION TITLE
Sub menu for opened audio applications for each device menu?

### DISCUSSION THREAD

**Discussion by gate1975mlm** [File-New-Project/EarTrumpet#665] [Ideas]
When you open EarTrumpet and if you have a lot of applications open with audio the list can get kind of high. So I was thinking how would it be if you added a sub menu button say next to the volume icon on each device menu? Or if not a button then perhaps a right click option on each device menu for the sub menu? And that way when you first click on the EarTrumpet icon in the system tray you will just see your default audio device menu like the standard Windows volume menu does.

Also if you do add this feature I would like for the applications sub menu to not be opened the next time you open EarTrumpet However if by chance someone does still like having the audio applications list showing the way it currently does then maybe you can also add a on and off option in the settings menu.

Would this be possible?

Thanks and keep up the great work EarTrumpet is a fantastic piece of software for our audio needs on Windows 10! :)

**Comment by nevcos** [comment]
An expand / collapse button on each device to show / hide the applications would be great.
A new setting would be needed as well to determine if the user wants all expanded (current behavior) or all collapsed by default.

Alternatively the current expand state could be persisted per device, so the UI would keep the same state between usages.
